core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -2742,31 +2734,19 @@ public function overrideAll()
 
 				try
 				{
+					$this->preloadCurrentRecords($ids);
+
 					foreach ($ids as $id)
 					{
-						try
-						{
-							$this->denyAccessUnlessGranted(ContaoCorePermissions::DC_ACTION_EDIT, new DataContainerSubject($this->strTable, $id));
-						}
-						catch (AccessDeniedException)
-						{
-							continue;
-						}
+						$currentRecord = $this->getCurrentRecord($id);
+
+						$this->denyAccessUnlessGranted(ContaoCorePermissions::DC_PREFIX . $this->strTable, new ReadAction($this->strTable, $currentRecord));
 
-						$this->intId = $id;
 						$this->procedure = array('id=?');
 						$this->values = array($this->intId);
 						$this->arrSubmit = array();
 						$this->blnCreateNewVersion = false;
 
-						// Get the field values
-						$objRow = $this->Database->prepare("SELECT * FROM " . $this->strTable . " WHERE id=?")
-												 ->limit(1)
-												 ->execute($this->intId);
-
-						// Store the active record
-						$this->objActiveRecord = $objRow;
```

Comment:
This should be restored to the current record?

Reply:
Jep, adjusted.